agu/_journals.tab

```diff
@@ -0,0 +1,21 @@
+Title ISSN  eISSN
+Earth and Space Science X 2333-5084
+Earth's Future  X 2328-4277
+Geochemistry, Geophysics, Geosystems  X 1525-2027
+Geophysical Research Letters  0094-8276 1944-8007
+Global Biogeochemical Cycles  0886-6236 1944-9224
+Journal of Advances in Modeling Earth Systems X 1942-2466
+Journal of Geophysical Research 0148-0227 2156-2202
+Journal of Geophysical Research: Atmospheres  2169-897  X 2169-8996
+Journal of Geophysical Research: Biogeosciences 2169-8953 2169-8961
+Journal of Geophysical Research: Earth Surface  2169-9003 2169-9011
+Journal of Geophysical Research: Oceans 2169-9275 2169-9291
+Journal of Geophysical Research: Planets  2169-9097 2169-9100
+Journal of Geophysical Research: Solid Earth  2169-9313 2169-9356
+Journal of Geophysical Research: Space Physics  2169-9380 2169-9402
+Paleoceanography  0883-8305 1944-9186
+Radio Science 0048-6604 1944-799X
+Reviews of Geophysics X 1944-9208
+Space Weather 1539-4956 1542-7390
+Tectonics 0278-7407 1944-9194
+Water Resources Research  0043-1397 1944-7973
```

Comment:
Yes me too. But there are tabs. I had twice delete it and use TAB key. Do you have any tool for controlling it?